toolchains/sysimage/BUILD.bazel

```diff
@@ -4,7 +4,7 @@ load("@rules_python//python:defs.bzl", "py_binary")
 package(default_visibility = ["//visibility:public"])
 
 exports_files([
-    "build_base_container_image.py",
+    "build_container_base_image.py",
```

Comment:
Huh. If the build was fine with an incorrect name, do these need to be exported at all?

Reply:
Not necessarily, since there is a build rule and others access only the build rule and not the file. I think in general it'd be better to use build rules instead of naked files but I didn't want to change everything. Lmk if you want me to remove the unnecessary exports.

Reply:
Let's leave it for this PR, we can come through and audit/cleanup later.